src/components/TouchCapture/src/TouchCapture.vue

```diff
@@ -0,0 +1,158 @@
+<template>
```

Comment:
This looks great, but I am curious if there's any small third-party packages that'd give us all this and more? I could see us doing a lot more with these touch events, particularly with other things we're seeing like carousel (swipes) and zooms (pinch).

Reply:
I poked around a bit and found a number of libraries but ultimately their benefits is mostly around a cleaner looking code. The downsides is we introduce an additional dependency, additional filesize, and hitching our wagon to something that isn't likely to get many updates later as the community is moving to Vue 3.

https://openbase.com/categories/js/best-vue-touch-events-libraries